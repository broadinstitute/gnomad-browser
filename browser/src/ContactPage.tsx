import React from 'react'

import { ExternalLink, PageHeading } from '@gnomad/ui'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'

export default () => (
  <InfoPage>
    <DocumentTitle title="Contact" />
    <PageHeading>Contact</PageHeading>

    <p>
      Tell us how you use gnomAD and your wish list by filling out{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="http://broad.io/2024_survey">our user survey</ExternalLink>.
    </p>

    <p>
      Use the gnomAD{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="https://discuss.gnomad.broadinstitute.org/new-topic?category=General">
        Forum
      </ExternalLink>{' '}
      to request help, discuss the data, and ask questions.*
    </p>

    <p>
      Report errors in the website on{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="https://github.com/broadinstitute/gnomad-browser/issues/new?labels=Type%3A%20Bug">
        GitHub
      </ExternalLink>{' '}
      or the{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="https://discuss.gnomad.broadinstitute.org/new-topic?category=Browser&tags=bug">
        Forum
      </ExternalLink>
    </p>

    <p>
      For questions about gnomAD, check out the{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="/help">help page</ExternalLink>.
    </p>

    <p>
      Note that, for many reasons (including consent and data usage restrictions), we do not have
      (and cannot share) phenotype information. Overall, we have limited information that we can
      share for some cohorts, such as last known age in bins of 5 years (when known) and chromosomal
      sex.
    </p>

    <p>
      Follow us on Twitter{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="https://twitter.com/gnomad_project">@gnomad_project</ExternalLink>.
    </p>

    <br />
    <p>
      *Alternately, you can{' '}
      {/* @ts-expect-error TS(2786) FIXME: 'ExternalLink' cannot be used as a JSX component. */}
      <ExternalLink href="mailto:gnomad@broadinstitute.org">email us</ExternalLink>. Please note
      that we prioritize answering issues on Github and topics on the Forum, so if you choose to
      email it may take us longer to respond.
    </p>
  </InfoPage>
)
