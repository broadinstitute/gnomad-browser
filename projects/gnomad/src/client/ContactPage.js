import React from 'react'

import { ExternalLink, PageHeading } from '@broad/ui'

import DocumentTitle from './DocumentTitle'
import InfoPage from './InfoPage'

export default () => (
  <InfoPage>
    <DocumentTitle title="Contact" />
    <PageHeading>Contact</PageHeading>
    <p>
      Errors in the website can be{' '}
      <ExternalLink href="https://github.com/macarthur-lab/gnomadjs/issues">
        reported via GitHub
      </ExternalLink>
      .
    </p>

    <p>
      You can also contact us by{' '}
      <ExternalLink href="mailto:exomeconsortium@gmail.com">email</ExternalLink> to report data
      problems or feature suggestions.
    </p>

    <p>
      Sign up for our low-volume mailing list for future release announcements{' '}
      <ExternalLink href="https://groups.google.com/forum/#!forum/exac_data_announcements">
        here
      </ExternalLink>
      .
    </p>
  </InfoPage>
)
