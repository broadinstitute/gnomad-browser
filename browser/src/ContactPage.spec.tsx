import { expect, test } from '@jest/globals'
import 'jest-styled-components'

import React from 'react'
import renderer from 'react-test-renderer'

import ContactPage from './ContactPage'
import { BrowserRouter } from 'react-router-dom'

test('Contact Page has no unexpected changes', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <ContactPage />
    </BrowserRouter>
  )
  expect(tree).toMatchSnapshot()
})
