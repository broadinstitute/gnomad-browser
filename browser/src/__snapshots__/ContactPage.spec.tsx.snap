// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Page has no unexpected changes 1`] = `
.c3 {
  color: #1173bb;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3:visited,
.c3:active {
  color: #1173bb;
}

.c3:focus,
.c3:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding-bottom: 0.5em;
  border-bottom: 1px solid #ccc;
  margin: 0.67em 0;
  font-size: 14px;
}

.c2 {
  margin: 0;
}

.c0 {
  box-sizing: border-box;
  width: 100%;
  max-width: 1200px;
  padding: 0 15px;
  margin: 0 auto 40px;
  font-size: 14px;
  font-size: 16px;
}

.c0 p {
  margin-bottom: 1em;
  line-height: 1.4;
}

@media (max-width:900px) {
  .c1 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    border-bottom: none;
  }
}

@media (max-width:900px) {
  .c2 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    padding-bottom: 0.5em;
    border-bottom: 1px solid #ccc;
    margin-bottom: 0.5em;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <h1
      className="c2"
    >
      Contact
    </h1>
  </div>
  <p>
    Tell us how you use gnomAD and your wish list by filling out
     
    <a
      className="c3"
      href="http://broad.io/2024_survey"
      rel="noopener noreferrer"
      target="_blank"
    >
      our user survey
    </a>
    .
  </p>
  <p>
    Use the gnomAD
     
    <a
      className="c3"
      href="https://discuss.gnomad.broadinstitute.org/new-topic?category=General"
      rel="noopener noreferrer"
      target="_blank"
    >
      Forum
    </a>
     
    to request help, discuss the data, and ask questions.*
  </p>
  <p>
    Report errors in the website on
     
    <a
      className="c3"
      href="https://github.com/broadinstitute/gnomad-browser/issues/new?labels=Type%3A%20Bug"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
    </a>
     
    or the
     
    <a
      className="c3"
      href="https://discuss.gnomad.broadinstitute.org/new-topic?category=Browser&tags=bug"
      rel="noopener noreferrer"
      target="_blank"
    >
      Forum
    </a>
  </p>
  <p>
    For questions about gnomAD, check out the
     
    <a
      className="c3"
      href="/help"
      rel="noopener noreferrer"
      target="_blank"
    >
      help page
    </a>
    .
  </p>
  <p>
    Note that, for many reasons (including consent and data usage restrictions), we do not have (and cannot share) phenotype information. Overall, we have limited information that we can share for some cohorts, such as last known age in bins of 5 years (when known) and chromosomal sex.
  </p>
  <p>
    Follow us on Twitter
     
    <a
      className="c3"
      href="https://twitter.com/gnomad_project"
      rel="noopener noreferrer"
      target="_blank"
    >
      @gnomad_project
    </a>
    .
  </p>
  <br />
  <p>
    *Alternately, you can
     
    <a
      className="c3"
      href="mailto:gnomad@broadinstitute.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      email us
    </a>
    . Please note that we prioritize answering issues on Github and topics on the Forum, so if you choose to email it may take us longer to respond.
  </p>
</div>
`;
